CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpMenuViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  HelpMenuViewController.swift
+//  GiniCapture
+//
+//  Created by Enrique del Pozo Gómez on 10/18/17.
+//  Copyright © 2017 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+/**
+ The `HelpMenuViewControllerDelegate` protocol defines methods that allow you to handle table item selection actions.
+ 
+ - note: Component API only.
+ */
+
```

Comment:
@chriskryn could you please add TODO - REMOVE Componen API because we will remove all related Component API code in the nearest future?